yutarour.md

```diff
@@ -61,5 +61,7 @@ I will be making a model of the enclosure in fusion 360 and printing it with a r
 | Isopropyl Alchohol        | Already Own                           | $0.00  |
 | 9V battery                | Already Own                           | $0.00  |
 | Female to Male wires      | Already Own                           | $0.00  |
-| Translucent Resin         | https://a.co/d/cN24vPx                | $17.99 |
-| Total                     |                                       | $248.98|
+| Translucent Resin         | Will purchase with own money          | $17.99 |
+| Tax (6.5%)                 |                                       | $15.01|
```

Comment:
Sales tax: 6.5%

Reply:
Amazon has already included the sales tax in their listing, but nothing bad about being safe with the budget